metadata/com.sparklinlabs.Superpowers.appdata.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.sparklinlabs.Superpowers.desktop</id>
```

Comment:
I actually ran `appstream-util validate-relax resources/com.sparklinlabs.Superpowers.appdata.xml` and it is it's OK. I didn't run neither validate nor validate-strict, because they are way too picky.